samtranslator/translator/verify_logical_id.py

```diff
@@ -1,3 +1,6 @@
+from ..model import Resource
```

Comment:
nit: we tend to use full path (makes recognition a little quicker and doesn't need changing if moving the file)